controls/V-71921.rb

```diff
@@ -19,6 +19,7 @@
   tag "cci": ["CCI-000196"]
   tag "documentable": false
   tag "nist": ["IA-5 (1) (c)", "Rev_4"]
+  tag "subsystems": ['useradd', 'password']
```

Comment:
This should be `login_defs` instead of `useradd`. The SIMP module name doesn't always align with the subsystem.